easybuild/easyconfigs/v/VSCode/VSCode-1681293081.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'Tarball'
+
+name = 'VSCode'
+version = '1681293081'
```

Comment:
@SimonPinches I wonder whether there's a friendly version we could add, e.g. the date, as well as this ID?